docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- KVM
+- firecracker (v0.21.0 or higher)
+- CNI installed to `/opt/cni` (`CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities are required)
+- iptables
+
+## Create the Cluster
+
+```bash
+talosctl cluster create --wait --provisioner firecracker
+```
+
+Once the above finishes successfully, your talosconfig(`~/.talos/config`) will be configured to point to the new cluster.
+
+> Note: You must run the above as `root`.
+
+## Retrieve and Configure the `kubeconfig`
+
+```bash
+talosctl kubeconfig .
+kubectl --kubeconfig kubeconfig config set-cluster talos-default --server https://127.0.0.1:6443
```

Comment:
I think we don't need to override the endpoint, default one should work (it points to bridge IP)

Reply:
Looks like 127.0.0.1 is still there

Reply:
Whoops. Thanks.

Reply:
Fixed.